### PR TITLE
[Oy2-19473] Create Logo Component

### PR DESCRIPTION
**Pull request by jphechter**
## Purpose

Create a new Logo component.

## Type of Request

feature

## Approach

Tried to follow existing patterns established in other components.

## Assorted Notes/Considerations

Because this component is dealing primarily with images, I included 2 new image files (an SVG and PNG of CMS.gov logo) to be used in the Storybook implementation. Additionally, Jest does not process images well, so there is a new transform module `imgTransform.js` that is called whenever Jest imports an SVG or PNG file. This is generic and can be extended to cover other file types.

There seems to be an import issue in both `Logo.test.tsx` and `Logo.stories.tsx`. I think this is just TypeScript being funny. Doesn't seem to be affecting anything (tests are green, no errors in app build), but I would like to figure this out.
Screen Shot 2022-07-27 at 4 41 21 PM

#### Pull Request Creator Checklist

- [X] It is documented what type of request this is.
- [X] There is a detailed description for the purpose of this PR.
- [X] This PR is in line with the intention of this Project

### CODE REVIEW

File: src/components/Logo/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  role = "img",
+  source,
+  width = 200,
+  ...rest
+}) => {
+  return (
+    <div
+      role={role}
+      aria-label={ariaLabel}
+      className={`application-logo-container ${className}`}
```
Comment (jdinh8124) [inline src/components/Logo/Logo.tsx:43]: Maybe some sort of check here in the change its undefined

`${className ??  ""}` or `${classname? className : ""}`

**Review by jdinh8124 — CHANGES_REQUESTED** [review]
Looks good for the most part, had some small comments

File: src/components/Logo/Logo.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    className: {
+      description:
+        "Determines whether or not the element is hidden from a screen reader.",
+    },
+    height: {
+      description: "Optional additional className for the logo container.",
+    },
+    role: {
+      description: "Height parameter for logo in # of pixels.",
+    },
```
[the remark this answers is not in the document]
Comment (jphechter) [reply src/components/Logo/Logo.stories.tsx:38]: 🤦‍♂️

**Review by jdinh8124 — APPROVED** [review]
Looks good to me!

**Pull request merged**